src/ui/popup.js

```diff
@@ -359,10 +400,10 @@ export default class Popup extends Evented {
             this._tip       = DOM.create('div', 'mapboxgl-popup-tip', this._container);
             this._container.appendChild(this._content);
             if (this.options.className) {
-                this.options.className.split(' ').forEach(name =>
-                    this._container.classList.add(name));
+                this.options.className.split(' ').forEach(name => {
+                    this._container.classList.add(name);
+                });
```

Comment:
Is this change necessary, or perhaps just something left over from a previous change that modified this and was then reverted?

Reply:
In previous commit i rewrite foreEach loop to one line function `this._container.classList.add(...name)`(or something like that), but i forgot to check browser compatibility – IE11 does not support multiple arguments, so i revert it to old good `forEach` to keep IE11 working. I noticed it [in this comment](https://github.com/mapbox/mapbox-gl-js/pull/8759#issuecomment-539913416)

Reply:
Thanks for the quick reply! Yeah I noticed the revert, I think just the brackets changed from the original (https://github.com/mapbox/mapbox-gl-js/blob/master/src/ui/popup.js#L367-L368) - not really a big deal but no point touching it if we don't need to.

Reply:
Without brackets function returns value, which is pointless for `forEach`. I think it's better use arrow function with brackets here. But as you said it's not a big deal, so i can remove it if you think that we should keep original variant.

Reply:
Yes exactly, all this does is change the return value of the function, which isn't used anyway. I don't see this change as necessary/an improvement but you can leave it in if you want to.